packages/edit-site/src/components/block-editor/editor-canvas.js

```diff
@@ -164,6 +164,7 @@ function EditorCanvas( {
 					'edit-site-visual-editor__editor-canvas',
 					{
 						'is-focused': isFocused && canvasMode === 'view',
+						'is-zoomed-out': isZoomOutMode,
```

Comment:
I think to me it makes more sense that we move the CSS to edit-site rather than keep them inside the block editor package, it's where the scaling is done, so the styling related to it should be in the same place, right? I'd appreciate any feedback there, it's not too clear to me.

Reply:
For me, ultimately, it's better to move the CSS, scaling and everything to "block-editor" or if we can't for some reason to "editor" because we know we want zoom-out for both edit-post and edit-site and we should stop building site editor specific features.

Reply:
ok, I read this after I pushed my last changes. The way I went about it is this:

- The scaling is happening to the canvas, which to me it isn't 100% clear it's the block editor or edit-site, but whatever it's inside, we care about that container being scaled and having a proper background color, that should happen all together
- Then we have what happens inside the canvas, and how it needs to change when it's scaled down, that I left alone how it was right now, but to achieve my first point I duplicated the is-zoomed-out class.

We are already styling the iframe differently if it's focused, or if it's a navigation block.

Reply:
I moved the CSS to the block editor package, but I still need the class to be in the iframe. Does that seem OK to you?